README.md

```
@@ -1,362 +1,141 @@
-| `CODEXMATE_FORCE_RESET_EXISTING_CONFIG` | `0` | Set `1` to force bootstrap reset of existing config |
+## Special Thanks
 
-## Tech Stack
+Special thanks to all contributors for their contributions to Codex Mate ❤️
 
-- Node.js
-- Vue.js 3 (Web UI)
-- Native HTTP server
-- `@iarna/toml`, `json5`
+<a href="https://github.com/SakuraByteCore/codexmate/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=SakuraByteCore/codexmate" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)